sdk/tests/conformance2/rendering/element-index-uint.html

```diff
@@ -89,7 +89,6 @@
         runIndexValidationTests(drawType);
         runCopiesIndicesTests(drawType);
         runResizedBufferTests(drawType);
-        runVerifiesTooManyIndicesTests(drawType);
```

Comment:
Removing runVerifiesTooManyIndicesTests is because that the body of it is almost same with runCopiesIndicesTests.